bin/propolis-server/src/lib/stats.rs

```diff
@@ -167,15 +166,22 @@ pub async fn start_oximeter_server(
     let producer_log = oximeter_producer::LogConfig::Logger(
         log.new(slog::o!("component" => "oximeter-producer")),
     );
+
+    // The maximum size of a single Dropshot request.
+    //
+    // This is a pretty arbitrary limit, but one that should be big enough for
+    // the statistics we serve today (vCPU usage and panic counts), with
+    // headroom for adding quite a few more.
+    const MAX_REQUEST_SIZE: usize = 1024 * 1024;
```

Comment:
looks like we also made this a lot bigger --- it was 2 KiB before, and now it's 1 MiB. was that intentional?